transaction-status/src/option_serializer.rs

```diff
@@ -0,0 +1,78 @@
+use serde::{ser::Error, Deserialize, Deserializer, Serialize, Serializer};
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum OptionSerializer<T: Default> {
```

Comment:
Is the `Default` bounds on `T` necessary anymore?

Reply:
Nope! I'll strip it out